### ISSUE TITLE
Composer autoload problem - due to php-sdk dep being hardcoded to 1.1.8

### ISSUE THREAD

**Issue by hostep** [dpdconnect/magento2-shipping#73]
Hi there

Using `dpdconnect/magento2-shipping` version 1.2.9 and `dpdconnect/php-sdk` 1.1.8

After installing it on a Magento shop and running a optimised autoload dump, I'm getting a warning:
```
$ composer dump-autoload -o
Generating optimized autoload files
Class DpdConnect\Sdk\Api\Data\ShipmentOrder\CustomsLinesInterface located in ./vendor/dpdconnect/php-sdk/src/Api/Data/ShipmentOrder/Customs/CustomsLinesInterface.php does not comply with psr-4 autoloading standard. Skipping.
```

This was fixed in https://github.com/dpdconnect/php-sdk/pull/21/files

However, we can't update to this version because the requirement is being hardcoded to version 1.1.8: https://github.com/dpdconnect/magento2-shipping/blob/f55297df96819f3803f22eaaa1cfdfb88537de91/composer.json#L6

Any reason why this can't be set to `^1.1.8`? Or was this an oversight?

_Update_, it looks like this used a more open constraint but that was removed in https://github.com/dpdconnect/magento2-shipping/pull/63 without a proper description telling us why?

**Comment by dpdplugin** [comment]
It has been locked to this version because the Magento module is not yet ready for the newest version of the SDK.
The newest version of the SDK requires a different implementation of caching.
We hope in the near future to be able to implement this. I cannot give you a date yet unfortunately.

**Comment by hostep** [comment]
Closing, this problem got fixed with the release of version 1.3.2 of this module, which now allows higher versions of the SDK again: https://github.com/dpdconnect/magento2-shipping/pull/91/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34
